blog/2024-11-21-unit-of-measurement-translations.md

```diff
@@ -0,0 +1,11 @@
+---
+author: Abílio Costa
+authorURL: https://github.com/abmantis
+title: "Translating units of measurement"
+---
+
+Home Assistant 2024.12 will support the translation of custom units of measurement. Previously, those would have been hardcoded in their integrations.
```

Comment:
```suggestion
Home Assistant 2024.12 will support the translation of custom unit of measurements. Previously, those would have been hardcoded in their integrations.
```

Reply:
I am not a native English speaker, but I think the plural of "unit of measurement" is "units of measurement".
Do you have any reference saying otherwise?

Reply:
Probably. But I think it looks a bit strange using it in plural form.

Reply:
The form in the PR is the right format. 👍